### ISSUE TITLE
Eliminate render-blocking JavaScript

### ISSUE THREAD

**Issue by Murderlon** [part-up/part-up#1000]
None of the above-the-fold content could be rendered without waiting for the following resources to load.

1. https://cdnjs.cloudflare.com/…ax/libs/js-cookie/2.1.3/js.cookie.min.js
2. https://cdnjs.cloudflare.com/…libs/dropbox.js/2.5.0/Dropbox-sdk.min.js
3. https://www.dropbox.com/static/api/2/dropins.js
4. https://apis.google.com/js/api.js
5. https://part-up.com/…d4191e3bdb68b.js?meteor_js_resource=true

### Possible solution
Concat and minify everything that isn't essential before the DOM, instead of requesting resources from multiple CDN's. Subsequently the `vendor.min.js` could have a `async` attribute to make it non-blocking. Concatenation is also preferable for users with HTTP/1.

**Comment by ralphboeije** [comment]
Same as with #1001 - good to asses the positive impact on Friday as performance is quite an issue.

**Comment by Murderlon** [comment]
@ralphboeije it is not quite the same as #1001, the advanced solution mentioned there is definitely an issue of its own.

**Comment by ralphboeije** [comment]
true, I meant to say that my comment was the same as with #1001 ;-)
